src/components/pages/main.js

```diff
@@ -1,559 +0,0 @@
-import React, { Component } from 'react'
```

Comment:
Just kind of curious, why was this deleted? It seems to be the same as Main.js, was there ever a difference between the two or just wanted to change the name?